suite-native/module-passphrase/src/components/PassphraseForm.tsx

```diff
@@ -78,6 +78,10 @@ export const PassphraseForm = ({ inputLabel, onFocus }: PassphraseFormProps) =>
         reset();
     });
 
+    const offerPassphraseOnDevice = !!device?.features?.capabilities?.includes(
+        'Capability_PassphraseEntry',
+    );
```

Comment:
Please get features using `selectDeviceFeatures` or better yet, create a selector for this.

Reply:
7fecf6c399